src/main/kotlin/com/quipalup/katydid/logentry/application/CreateLogEntryCommandHandler.kt

```diff
@@ -0,0 +1,26 @@
+package com.quipalup.katydid.logentry.application
+
+import arrow.core.Either
+import com.quipalup.katydid.logentry.domain.LogEntryError
+import com.quipalup.katydid.logentry.domain.LogEntryRepository
+import com.quipalup.katydid.logentry.logentry.primaryadapter.httprest.LogEntryRequestDocument
+import com.quipalup.katydid.logentry.logentry.primaryadapter.httprest.LogEntryResourceAttributes
+import com.quipalup.katydid.logentry.logentry.primaryadapter.httprest.LogEntryResponseDocument
+import javax.inject.Named
+
+@Named
+class CreateLogEntryCommandHandler(private val logEntryRepository: LogEntryRepository) {
+    fun execute(createLogEntryByCommand: CreateLogEntryCommand): Either<LogEntryError, LogEntryResponseDocument> {
+        return createLogEntryByCommand.toCreateRequest().let {
+            logEntryRepository.create(it)
+        }
+    }
+}
+
+private fun CreateLogEntryCommand.toCreateRequest(): LogEntryRequestDocument {
+    return LogEntryRequestDocument(data = CreateLogEntryCommand(this.type, this.attributes))
+}
+    data class CreateLogEntryCommand(
+        val type: String,
+        val attributes: LogEntryResourceAttributes
+    )
```

Comment:
```suggestion
    data class CreateLogEntryCommand(
    val time: String,
    val description: String,
    val amount: Number,
    val unit: String
    )
```
We don't need to replicate data structures through all layers. In the current state of this use case, this fields will be enough.